src/components/user-steps-wrapper/UserStepsWrapper.jsx

```diff
@@ -56,7 +56,12 @@ const UserStepsWrapper = ({ userRole }) => {
           <StepProvider initialValues={initialValues} stepLabels={stepLabels}>
             <StepWrapper steps={stepLabels}>{childrenArr}</StepWrapper>
           </StepProvider>
-        )
+        ),
+        dialogConfig: {
+          title: 'Please Confirm',
+          message:
+            'Are you certain you want to close? Any unsaved changes will be lost'
```

Comment:
Please move this string in `.json` file in `constants` folder.

Reply:
@vlad-khrychov changed messages to the predefined ones